core-bundle/src/Resources/contao/library/Contao/Controller.php

```diff
@@ -1625,7 +1625,7 @@ public static function addImageToTemplate($template, array $rowData, $maxWidth =
 			}
 
 			$size[0] = $maxWidth;
-			$size[1] = floor($maxWidth * ($height / $width));
+			$size[1] = (int) floor($maxWidth * ($height / $width));
```

Comment:
I noticed this inconsistency when writing the new tests. Turns out `floor` returns a `float` :facepalm:, so the size array looked like `[100, 75.0, 'crop']`. Apparently did not cause problems but isn't correct either.